meteor/client/lib/triggers/TriggersHandler.tsx

```diff
@@ -304,11 +306,15 @@ export const TriggersHandler: React.FC<IProps> = function TriggersHandler(
 		]
 		const systemActionKeys = ['Enter', 'NumpadEnter', 'Tab']
 
+		const poisonKey: string | null = Meteor.settings.poisonKey
```

Comment:
In the way this is currently set up, this needs to import `Settings` from `meteor\lib\Settings.ts` and use `Settings.poisonKey` as that's where the "defaulted" version of settings lives.

Reply:
@jstarpl Thanks for the catch. It is fixed now.